common/djangoapps/util/milestones_helpers.py

```diff
@@ -0,0 +1,136 @@
+"""
+Helper methods for milestones api calls.
+"""
+
+from collections import defaultdict
+from opaque_keys.edx.keys import CourseKey
+from milestones.api import (
+    get_course_milestones,
+    get_courses_milestones,
+    get_user_milestones,
+    add_milestone,
+    add_course_milestone,
+    remove_course_milestone,
+    get_course_milestones_fulfillment_paths,
+)
+from django.conf import settings
+
+
+def get_prerequisite_course_key(course_key):
+    """
+    Retrieves pre_requisite_course_key for a course from milestones app
```

Comment:
It looks like this currently returns just the first prerequisite course (if any exist). Update the documentation accordingly?